.plzconfig

```diff
@@ -1,6 +1,7 @@
 ; Please config file
 [please]
 version = 16.27.2
+VersionChecksum = 91c6ff81c53bbd105133306faa977f59da1b72bf28e0ecb7fe0778529c5cd861
```

Comment:
Mind adding the versionchecksum for ARM64 as well given #4 ?

Reply:
definitely! as part of the arm64 support pr :D